### ISSUE TITLE
Reports: Empty filters in reports pages

### ISSUE THREAD

**Issue by myriamboure** [openfoodfoundation/openfoodnetwork#2212]
## Description
**- As a:** enterprise manager

**- On page:** /admin/reports when clicking on any link from that page
![screenshot from 2018-04-12 19-35-30](https://user-images.githubusercontent.com/8135376/38693949-b68cd540-3e88-11e8-8779-c2ef3ff83b8a.png)

**- I want to be able to do:** access the page (no data loaded already done) with no pre-defined filters options (= empty filters)

## Acceptance criteria
1- When the user click on any of the links from the /admin/reports, the page as described in the precedent issue #2211 opens, but now **without** any filter.
![untitled presentation 1](https://user-images.githubusercontent.com/8135376/38926264-5fa34aa6-4302-11e8-92b4-4c9f57c4b46e.jpg)

2- Once the user applies filters and click on "search", the page reloads, but does not open a new window (it is already the case).

**Comment by myriamboure** [comment]
Then @RachL this one goes just after/simultaneously with the other story, awesome if you can work on UX for this as well :-)

**Comment by RachL** [comment]
@myriamboure @mkllnk I guess my comment in #2211 is enough for the user, except if we can add shortcuts for filters that are often used. I think you've mentionned that the last order cycle was often asked, am I correct ? But maybe it goes beyond the scope of this story and we need another issue for this.

**Comment by myriamboure** [comment]
Yes @RachL let's just here remove all filers and display the message you suggest, that should be fine :-)

**Comment by myriamboure** [comment]
I think @RachL suggesting default queries are beyond the scope of the focus point we decided to prioritize, we can work on that when we decide to focus on imprpving UX of back office (there are many UX twicks in the list already!)

**Comment by myriamboure** [comment]
Same here @mkllnk I put it in your queue, after #2211 :-)

**Comment by myriamboure** [comment]
@mkllnk the problem is that we have many users who reported that they made mistakes when they had order cycle in between two months, as the default start date was "first day of the current month" so when they go on the 3rd month X to extract data they select their OC but forget to change the start date. This is very confusing.
We also have added the small indication text driving the user in what she is supposed to do: "Select filters and click on "search" in order to access your data".
That's the explanation.
#2211 was "don't load any data", this one is "remove default filters".
Is that clearer?
